addons/stock/views/stock_picking_views.xml

```diff
@@ -143,7 +143,14 @@
                             invisible="not has_scrap_move"/>
                         <button name="action_see_packages" string="Packages" type="object"
                             class="oe_stat_button" icon="fa-cubes"
-                            invisible="not has_packages"/>
+                            invisible="state == 'done'">
+                            <field name="packages_count" widget="statinfo" string="Packages"/>
+                        </button>
+                        <button name="action_see_package_histories" type="object"
+                            class="oe_stat_button" icon="fa-cubes"
+                            invisible="state != 'done'">
+                            <field name="packages_count" widget="statinfo" string="Packages"/>
```

Comment:
Could maybe be only one action that return a dict depending on the picking state

Reply:
I thought it was clearer to keep them separate, but yes. As you prefer.